src/app/[locale]/dataset-list.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import {useQuery} from '@tanstack/react-query';
+import {SearchResult} from '@/lib/dataset-fetcher';
+import DatasetCard from '@/components/dataset-card';
+
+interface Props {
+  initialSearchResult: SearchResult;
+  locale: string;
+}
+
+export default function DatasetList({initialSearchResult, locale}: Props) {
+  const query: {data: SearchResult} = useQuery({
```

Comment:
I agree. I thought this was a hard one to name. I would not want to call it results, because I want It to be clear it's something else than searchResult. `query` is a name used in the react-query docs. But I agree on this is not really a query. So I will call it `response`

Reply:
👍🏻